README.md

```diff
@@ -256,25 +256,20 @@ In general, every development board has the following basic features:
 - [x] Modular testing of the circuit
 - [x] Testing of prototype board
 - [x] Final version
+- [x] Making the board Battery Compatible
 - [ ] Resolving high frequency noise by BO Motors ( TB6612FNG )
 
 <!-- CONTRIBUTORS -->
 ## Contributors
 
-- [Chinmay Lonkar](https://github.com/ChinmayLonkar): *Designer*
-- [Dhairya Shah](https://github.com/dhairyashah1): *Mentor*
-<!-- - [Omkar Bhilare](https://github.com/ombhilare999): *Mentor*
-- [VedantParanjape](https://github.com/VedantParanjape): *Mentor*
```

Comment:
Add a small colapsable toggle to acknowledge all previous contributors.

https://gist.github.com/Ashton-W/864d09d71f7cb8e5b47ebaac02b98455